test_with_svm.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+pushd ./vendor/github.com/ethereumproject/evm-ffi/c/
```

Comment:
`make -C ./vendor/github.com/ethereumproject/evm-ffi/c/` instead of `pushd`/`popd`